includes/class-signature.php

```diff
@@ -49,32 +40,51 @@ public static function get_public_key( $user_id, $force = false ) {
 	 *
 	 * @return mixed The private key.
 	 */
-	public static function get_private_key( $user_id, $force = false ) {
+	public static function get_private_key_for( $user_id, $force = false ) {
 		if ( $force ) {
```

Comment:
I think we can get rid of the `$force` argument here, since `get_keypair_for` is already generating the key if one isn't found.